test/parallel-benchmark/mzcompose.py

```diff
@@ -0,0 +1,1190 @@
+                        ClosedLoop(
+                            action=StandaloneQuery(
+                                "SELECT * FROM mv_sum_mysql",
+                                conn_info=conn_infos["materialized"],
+                                strict_serializable=True,
+                            ),
+                        )
+                        for i in range(10)
+                    ],
+                ),
+            ],
+            guarantees={
+                "SELECT * FROM mv_sum_mysql (standalone)": {"qps": 15, "max": 4000},
+            },
+        )
+
+
+class OpenIndexedSelects(Scenario):
+    def __init__(self, c: Composition, conn_infos: dict[str, PgConnInfo]):
+        self.init(
+            [
+                TdPhase(
+                    """
+                    > CREATE TABLE t4 (f1 TEXT, f2 INTEGER);
+                    > CREATE DEFAULT INDEX ON t4;
+                    > INSERT INTO t4 VALUES ('A', 1);
+                    > INSERT INTO t4 VALUES ('B', 2);
+                    > INSERT INTO t4 VALUES ('C', 3);
+                    > INSERT INTO t4 VALUES ('D', 4);
+                    """
+                ),
+                LoadPhase(
+                    duration=120,
+                    actions=[
+                        OpenLoop(
+                            action=PooledQuery(
+                                "SELECT * FROM t4", conn_info=conn_infos["materialized"]
+                            ),
+                            dist=Periodic(per_second=400),
+                        ),
+                    ],
+                ),
+            ],
+            conn_pool_size=100,
+            guarantees={
+                "SELECT * FROM t4 (pooled)": {"qps": 200, "max": 150},
+            },
+        )
+
+
+class ConnectRead(Scenario):
+    def __init__(self, c: Composition, conn_infos: dict[str, PgConnInfo]):
+        self.init(
+            [
+                LoadPhase(
+                    duration=120,
+                    actions=[
+                        ClosedLoop(
+                            action=StandaloneQuery(
+                                "SELECT 1",
+                                conn_info=conn_infos["materialized"],
+                                strict_serializable=False,
+                            ),
+                        )
+                        for i in range(10)
+                    ],
+                ),
+            ],
+            guarantees={
+                "SELECT * FROM t4 (pooled)": {"qps": 35, "max": 700},
+            },
+        )
+
+
+class FlagUpdate(Scenario):
+    """Reproduces #29235"""
+
+    def __init__(self, c: Composition, conn_infos: dict[str, PgConnInfo]):
+        self.init(
+            [
+                LoadPhase(
+                    duration=120,
+                    actions=[
+                        OpenLoop(
+                            action=ReuseConnQuery(
+                                "ALTER SYSTEM SET enable_table_keys = true",
```

Comment:
<s>What is the benefit of running this multiple times? Or was the intention to flip the flag?</s>
Understood after looking at the referenced issue.